### PR TITLE
Fix on_body issue for Typhoeus

### PR DESCRIPTION
**Pull request by sazor**

### CODE REVIEW

**Comment by mcfiredrill** [comment]
Hi, Thanks for the PR. Tests are failing, but not sure if they are related to your change yet. 
I'll try to take a closer look at this when I have some time.

**Comment by sazor** [comment]
Hi, @mcfiredrill. Thank you. These tests were failing even before i started to do something. Maybe I'll take a look at them too and try to figure out whats wrong.

**Comment by sazor** [comment]
They're failing because of some other issue.

**Comment by ecnalyr** [comment]
@sazor Could you rebase, please?

Travis will still be red, but master is passing on four out of five of the current builds.

If we get this branch passing on those same four builds we should be able to merge it.

File: lib/vcr/library_hooks/typhoeus.rb [hunk 1]
```diff
@@ -117,4 +135,3 @@ def self.vcr_response_from(response)
     WebMock::HttpLibAdapters::TyphoeusAdapter.disable!
   end
 end
-
```
Comment (ecnalyr) [inline lib/vcr/library_hooks/typhoeus.rb]: I think the convention for VCR is to have a newline at the end of every file, could you re-introduce these lines, please?

File: spec/lib/vcr/library_hooks/typhoeus_spec.rb [hunk 2]
```diff
@@ -159,4 +177,3 @@ def make_single_request(path, options = {})
     end
   end
 end
-
```
Comment (ecnalyr) [inline spec/lib/vcr/library_hooks/typhoeus_spec.rb]: Same newline issue as above.

File: lib/vcr/library_hooks/typhoeus.rb (hunk 1 above)
Comment (ikapelyukhin) [inline lib/vcr/library_hooks/typhoeus.rb]: Would you consider using Rubocop to codify & automatically perform such code style checks?

Comment (ikapelyukhin) [inline lib/vcr/library_hooks/typhoeus.rb]: @ecnalyr ^^^

**Comment by ikapelyukhin** [comment]
@sazor That was quite an ugly rebase :-)

**Comment by sazor** [comment]
@ikapelyukhin Yeah, sorry, admit that. Quite old pull request, will rework it today.

**Comment by ikapelyukhin** [comment]
@sazor There's no need, I've already done that in #657 (and fixed couple of other issues, too).
Пыщь! :-)

**Comment by ikapelyukhin** [comment]
Err, I mean in #656.

**Comment by sazor** [comment]
Ah, ok. Then i should close this one, right?

**Comment by ikapelyukhin** [comment]
@sazor Yeah, I guess.

**Pull request closed without merging**